packages/components/src/components/cookies-consent.tsx

```diff
@@ -0,0 +1,47 @@
+import { useCallback, useState } from 'react';
+import { cn } from '../cn';
+import { CallToAction } from './call-to-action';
+
+export interface CookiesConsentProps extends React.HTMLAttributes<HTMLElement> {}
+export function CookiesConsent(props: CookiesConsentProps) {
+  const [show, setShow] = useState(() => localStorage.getItem('cookies') !== 'true');
+
+  const accept = useCallback(() => {
+    setShow(false);
+    localStorage.setItem('cookies', 'true');
+  }, []);
+
+  if (!show) {
+    return null;
+  }
+
+  return (
+    <div
+      {...props}
+      className={cn(
+        'fixed bottom-0 z-50 flex flex-wrap items-center justify-center gap-x-4 gap-y-3 rounded-lg border border-beige-200 bg-beige-100 p-4 text-sm text-green-800 shadow-xl lg:flex-nowrap lg:justify-between lg:text-left dark:border-neutral-800 dark:bg-neutral-900 dark:text-neutral-200',
+        props.className,
+      )}
+    >
+      <div>
```

Comment:
```suggestion
      <div className="max-sm:inline">
```
set in parent and remove from its children